### DISCUSSION TITLE
Welcome to protocol-core-v1 Discussions!

### DISCUSSION THREAD

**Discussion by LeoHChen** [storyprotocol/protocol-core-v1#179] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AbdulrazaqAS** [comment]
Hello, I'm Abdulrazaq. I started learning Blockchain earlier this year with Alchemy University. I've finished a coupIe of courses and I now want to try something big. As a participant in the previous Story's buildathon, I decided to study the protocol because I loved its concept.

I started with the registry module and I'm learning a lot, especially design patterns and EIPs. I'm amazed by the design of the account storage contract (how it's namespaced) and I'm now adopting that in my upgradeable contracts.

Hope I'll get the support because I'm getting questions that I should ask. Please is someone still here? Pls respond.
